### PR TITLE
fix: insights edit repo list limit

### LINKED ISSUE #1751: Bug: Insights page more than 10 repos obscured when editing page
### Describe the bug

I have a page of 30 repos:

Screenshot 2023-09-26 at 1 30 37 PM

But when I go to edit the insight page, I only see the first 10

Screenshot 2023-09-26 at 1 31 02 PM

My initial reaction was that something went wrong and one of my previous edits to this page didn't go through correctly. I ended up going back to the main overview page to confirm that this wasn't the case: it had in fact correctly updated the page. It just wasn't displaying all the repos.

### Steps to reproduce

1. Go to an insights page with over 10 repos
2. Note the number of repos in the overview page
3. Edit the page
4. Note that only 10 are displayed

cc @isabensusan - more feedback: I'm not sure what the intent might be here for editing pages since new pages get dynamically added to the list during creation.

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by OgDev-01**
## Description
This PR adds a temporary fix to enable users see up to 50 repos when editing a list. This also adds a max width to the cart item wrapper with an over-flow scroll to enable users scroll through a large list without spoiling.g the experience

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
fixes #1751 

### Before 
image

### After
https://github.com/open-sauced/app/assets/62995161/8a7f5a52-02a3-41be-a733-4fefd2f27fba

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: pages/hub/insights/[insightId]/edit.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const EditInsightPage: WithPageLayout = () => {
   const { data: insight, isLoading: insightLoading, isError: insightError } = useInsight(id);
   const { isLoading: insightTeamMembersLoading, isError: insightTeamMembersError } = useInsightMembers(Number(id));
   const insightRepos = insight?.repos.map((repo) => repo.repo_id);
-  const { data: repos } = useRepositories(insightRepos);
+  const { data: repos } = useRepositories(insightRepos, 30, 50);
```
Comment (nickytonline) [inline pages/hub/insights/[insightId]/edit.tsx:20]: Any reason for using `50`? Could we not set it to the amount of repos that are associated with that insight page instead?

For future, we could add a load more... button or paging. cc: @isabensusan

Comment (OgDev-01) [reply pages/hub/insights/[insightId]/edit.tsx:20]: We have no clue what the amount is until the data is fetched from the hook. Setting 50 is a temporary fix as I'm sure not many users will have more than that amount of repos for now. We will definitely improve this with a load more paging approach in the future.

Comment (brandonroberts) [reply pages/hub/insights/[insightId]/edit.tsx:20]: That's correct. 50 is our upper limit at the moment also regardless of type of user

Comment (nickytonline) [reply pages/hub/insights/[insightId]/edit.tsx:20]: Sounds good!

**Review by nickytonline — APPROVED** [review]
🚀

**Review by brandonroberts — APPROVED** [review]

File: pages/hub/insights/[insightId]/edit.tsx (hunk 1 above)
Comment (jpmcb) [reply pages/hub/insights/[insightId]/edit.tsx:20]: I'm also curious why 50? I think ideally, like Nick said, I would anticipate it loading all. What if i have 51 repos in an insight page? Would the 1 repo that doesn't get included in the list to edit?

We should start thinking about the user flows for people who are going to have alot of repos (for example, there are hundreds of projects in Kubernetes alone and even more in the CNCF landscape)

Comment (jpmcb) [reply pages/hub/insights/[insightId]/edit.tsx:20]: Just seeing the above comments 👍🏼 if the soft limit is 50, let's proceed!

**Review by jpmcb — APPROVED** [review]

**Pull request merged**